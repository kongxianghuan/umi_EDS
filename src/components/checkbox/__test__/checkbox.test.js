import React from 'react'
import { mount } from 'enzyme'
import Checkbox from '..'

describe('Checkbox', () => {
  it('render correctly', () => {
    const checkbox = mount(<Checkbox>Checkbox</Checkbox>)   
    checkbox.setProps({ indeterminate: true })
    expect(checkbox.getDOMNode().indeterminate).toBeTruthy()
    expect(checkbox).toMatchSnapshot()
  })
})