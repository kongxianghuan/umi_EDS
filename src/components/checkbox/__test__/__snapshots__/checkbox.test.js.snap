// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox render correctly 1`] = `
<Checkbox
  indeterminate={true}
>
  <input
    id="eds-checkobx-1"
    type="checkbox"
  />
  <label
    htmlFor="eds-checkobx-1"
  >
    Checkbox
  </label>
</Checkbox>
`;
